doc/src/content/xdocs/gettingstartedpython.xml

```diff
@@ -51,8 +51,15 @@
     <section id="download_install">
       <title>Download</title>
       <p>
-        Avro implementations for C, C++, C#, Java, PHP, Python, and Ruby can be
-        downloaded from the <a
+        For Python, the easiest way to get started is to install it from PyPI.
+        Python's Avro API is available over <a href="https://pypi.org/project/avro/">PyPi</a>.
+      </p>
+      <source>
+$ pip install avro
```

Comment:
Ah, shoot, sorry I missed this, we should probably do it the same as below with `python3 -m pip`.

Reply:
No worries, I've updated this one as well.